src/pages/createlist/CreateListPage.test.tsx

```diff
@@ -93,7 +93,7 @@ describe('CreateList Page', () => {
     });
   });
 
-  describe('interactions', () => {
+  test.skip('interactions', () => {
```

Comment:
Please add a comment referencing the JIRA number so why know why this is being skipped now.